src/style/index.css

```diff
@@ -2,6 +2,8 @@
 @tailwind utilities;
 @tailwind components;
 
+@import url('https://fonts.googleapis.com/css2?family=Kanit:wght@300&family=Montserrat:wght@300&family=Roboto:ital,wght@1,700&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap');
```

Comment:
Do not use Google Fonts, use default fonts from Tailwind - it's because they are installed on every device. Remove this line